circle.yml

```diff
@@ -5,8 +5,11 @@ machine:
     # FIXME: Disabled $JRUBY_VERSIONS tests because of a Java incompatibility
```

Comment:
not really part of the pr, but I really think jruby should be supported (and should be much easier with 1.9 support).